settings.talon

```diff
@@ -27,6 +27,9 @@ settings():
     # If `true`, stop continuous scroll/gaze scroll with a pop
     user.mouse_enable_pop_stops_scroll = true
 
+    # If `true`, stop mouse drag with a pop
+    user.mouse_enable_pop_stops_drag = true
```

Comment:
I made the decision to enable this as default here sins the above `mouse_enable_pop_stops_scroll` already does that